src/components/layouts/joblogger/JobLogger.js

```diff
@@ -82,6 +82,11 @@ const JobLogger = ({ visible, onClose, onStatusChange }) => {
         const { id, finishedAt, exportUrl } = job;
         if (id !== jobId) return job;
 
+        if (job.state === JobStates.FAILED) {
+          // Jobs can't transition from a failed status to another status.
+          return job;
+        }
+
```

Comment:
```suggestion
        const { id, finishedAt, exportUrl, state } = job;
        if (id !== jobId) return job;

        if (state === JobStates.FAILED) {
          // Jobs can't transition from a failed status to another status.
          return job;
        }

```

Reply:
I've committed a similar change but with a smaller diff than the one in your suggestion